CONTRIBUTING.md

```diff
@@ -12,45 +12,57 @@ Pro tip: Use the table of contents in the top-left corner to jump to specific se
 
 Is this your first time working on open source projects? Check out this video [series](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github).
 
-## Contributing to our project (for freelancers)
+## Contributing to our project
 
-### Creating Issues
+You can contribute to this project as a team member or as a freelancer.  The instructions will indicate if an instruction is specific to only a team 
+member or only a facilitators with (team member only) or (facilitator only) before the instruction.
```

Comment:
Did you mean freelancer rather than facilitators? Freelancer is used further below while facilitator is not.